utils/feature_selection.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+from plotly.graph_objs._figure import Figure
```

Comment:
tuto idea pinda, ze veci, ktore sa zacinaju _ su podla konvencie Pythonu private a nemali by sa takto pouzivat, na tento ucel je to vsak asi jedno, kedze toto neni nas produkcny kod.

Reply:
prerobil som